### ISSUE TITLE
multi site ?

### ISSUE THREAD

**Issue by jonathanfidi** [netlify-templates/victor-hugo#219]
Is there a way to do multisite with Victor Hugo ?

**Comment by erquhart** [comment]
I don't believe that Hugo itself has a multisite feature per se - I believe it's possible, just not something they have special documented support for.
